jscomp/others/js_bigint.res

```diff
@@ -1,3 +1,74 @@
 /*** JavaScript BigInt API */
 
-type t
+@val
+/**
+Parses the given `string` into a `bigint` using JavaScript semantics. Return the
+number as a `bigint` if successfully parsed, `null`, `undefined`, `_NaN` otherwise.
+
+## Examples
+
+```rescript
+/* returns 123n */
+Js.Bigint.fromString("123")
+
+/* returns 0n */
+Js.Bigint.fromString("")
+
+/* returns 17n */
+Js.Bigint.fromString("0x11")
```

Comment:
So this is not supported as literal, but can be constructed in this way via parsing right?